src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1353,6 +1353,13 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
+    // TODO: Use `alt: 'media'` to download media content.
```

Comment:
On closer look, while CORS is documented as supported with the Javascript client library, it is not with the Node.js client library, so I'm going to leave this part out unless we have some positive indication for it.